src/graph_notebook/loader/load.py

```diff
@@ -34,21 +40,26 @@
                        'LOAD_FAILED_BECAUSE_DEPENDENCY_NOT_SATISFIED',
                        'LOAD_FAILED_INVALID_REQUEST', ]
 
+def do_load(host, port, load_format, use_ssl, source, region, arn, mode, fail_on_error, parallelism,
```

Comment:
can we go ahead and add default values for params that are optional? 
https://docs.aws.amazon.com/neptune/latest/userguide/load-api-reference-load.html#load-api-reference-load-syntax

Reply:
Done